url6da19b5a7a_krakrak_Adventure_Game_tgz-pJ8-adventure_MainJ8/src/NPC/Skjor.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * To change this template, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package NPC;
+
+import commands.Game;
+import commands.GameState;
+import items.FriedChicken;
+import items.IronCompanionsSword;
+import items.Item;
+import items.Key;
+import items.SilverCompanionsSword;
+import items.UnlockSpellBook;
+
+/**
+ *
+ * @author Irakiza Krak
+ */
+public class Skjor extends AbstractNpc{
+    private Item ironCompanionsSword = null;
+    
+     public Skjor(){
+        super("Skjor","Short and muscular man with a red sign of the Companions on his surcoat", "Finally here you are, here a mission for you : ", "Skj");
+        
+    }
+    @Override
+    public GameState action(Game game){
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

The cyclomatic complexity of this method is 11. By comparison, 98% of the methods in the CodeGuru reference dataset have a lower cyclomatic complexity. This indicates the method has a high number of decisions and it can make the logic difficult to understand and test. We recommend that you simplify this method or break it into multiple methods.